social-rules/index.html

```diff
@@ -69,14 +69,14 @@ <h3 id="backseat-driving">No backseat driving</h3>
 
         <h3 id="subtle-isms">No subtle -isms</h3>
-        <p>Subtle -isms are subtle expressions of racism, sexism, ageism, homophobia, transphobia and other kinds of bias and prejudice. They are small things that make others feel unwelcome, things that we all sometimes do by mistake. Subtle -isms make people feel like they don&rsquo;t belong at Ones and Zeros. We want to create an environment where everyone can focus all their energy on programming. It&rsquo;s hard to do that if you&rsquo;re regularly being made to wonder whether you belong.</p>
+        <p>Subtle -isms are subtle expressions of racism, sexism, ageism, homophobia, transphobia and other kinds of bias and prejudice. They are small things that make others feel unwelcome, things that we all sometimes do by mistake. Subtle -isms make people feel like they don&rsquo;t belong at Ones and Zeros. We want to create an environment where everyone can focus all their energy on learning. It&rsquo;s hard to do that if you&rsquo;re regularly being made to wonder whether you belong.</p>
         <p>Subtle -isms can also be things that you do instead of say. This includes things like boxing out the only woman at the whiteboard during a discussion or assuming someone isn&rsquo;t a programmer because of their race or gender.</p>
         <p>The fourth social rule is more complicated than the others. Not everyone agrees on what constitutes a subtle -ism. Subtle -isms are baked into society in ways that can make them hard to recognize. And not everyone experiences subtle -isms in the same way: subtle homophobia won&rsquo;t hurt someone who&rsquo;s straight in the same way it hurts someone who&rsquo;s gay.</p>
-        <p>There&rsquo;s another part of <i>no subtle -isms</i>: If you see racism, sexism, etc. outside of Ones and Zeros, please don&rsquo;t bring it in. For example, please don&rsquo;t start a discussion about the latest offensive comment from Random Tech Person Y. Everyone who comes to Ones and Zeros should have the same opportunity to focus on programming, and people from oppressed groups often find discussions of racism, sexism, etc. particularly hard to tune out. There are many places to discuss and debate these issues, but there are few where people can avoid them. Ones and Zeros is one of those places.</p>
+        <p>There&rsquo;s another part of <i>no subtle -isms</i>: If you see racism, sexism, etc. outside of Ones and Zeros, please don&rsquo;t bring it in. For example, please don&rsquo;t start a discussion about the latest offensive comment from Random Tech Person Y. Everyone who comes to Ones and Zeros should have the same opportunity to focus on learning, and people from oppressed groups often find discussions of racism, sexism, etc. particularly hard to tune out. There are many places to discuss and debate these issues, but there are few where people can avoid them. Ones and Zeros is one of those places.</p>
```

Comment:
this change was an accident and I don't know what it actually is :man_shrugging:

Reply:
wait, I'm not sure I get this comment -- isn't this what the PR is about?

Reply:
ooh I think maybe it's a github bug? looks like it's quoting a different part in the PR conversation view.

Reply:
k checked it out on my local git: there is a programming -> learning change in three paragraphs including this one, but for some reason github isn't highlighting the words on this one, and generally seeming all kinds of weird 🤷

Reply:
ohhh, good catch!